config/database.yml

```diff
@@ -34,6 +34,8 @@ default: &default
   adapter: postgresql
   url: <%= ENV["DATABASE_URL"] %>
   <% end %>
+  variables:
+    statement_timeout: <%= ENV.fetch("POSTGRES_STATEMENT_TIMEOUT", 300_000.seconds) %>
```

Comment:
should this be added to .env.example and app.json?

Reply:
Yes, good call!